ingester/database.py

```diff
@@ -133,8 +292,34 @@ def connect_datasheet_mc(self, datasheet_id, mc_id):
             query = """
                                 MATCH (ds:Datasheet {external_id: $data_id}), (mc:ModelCard {external_id: $mc_id})
                                 CREATE (mc)-[:TRAINED_ON]->(ds)
-                                """
-            session.run(query, data_id=datasheet_id, mc_id=mc_id)
+                                RETURN ds
+                                LIMIT 1
+                      """
+
+            result =  session.run(query, data_id=datasheet_id, mc_id=mc_id)
+            matched_node = result.single()
+
+            if not matched_node or not matched_node.get('ds'):
+                query = """
+                                   MATCH (mc:ModelCard {external_id: $mc_id})
+                                   CREATE (d:Datasheet {
+                                       external_id: $data_id,
+                                       name: 'Default Datasheet',
+                                       description: 'No description available',
+                                       source: 'Unknown source',
+                                       download_url: 'No URL provided',
+                                       version: '1.0',
+                                       license: 'Unknown license',
+                                       doi: 'No DOI provided',
+                                       target_variable: 'None',
+                                       categories: [],
+                                       datapoints: 0,
+                                       missing_values: 0,
+                                       attribute_types: []
+                                   })
+                                   CREATE (mc)-[:TRAINED_ON]->(d)
```

Comment:
Do we need to have these parameters there by default? We can't do updates on the datasheets. Shouldn't it be just the datasheet id and rest of it should come later?